content/en/account_management/rbac/permissions.md

```diff
@@ -0,0 +1,372 @@
+---
+title: Datadog Role permissions
+kind: documentation
+aliases:
+  - /account_management/faq/managing-global-role-permissions
+further_reading:
+    - link: '/account_management/rbac/'
+      tag: 'Documentation'
+      text: 'Learn how to create, update and delete a Role'
+    - link: '/api/#get-permissions'
+      tag: 'Documentation'
+      text: 'Manage your permissions with the Permission API'
+---
+
+Once your roles are created, assign or remove permission to this role directly by [updating the role in the Datadog application][1], or through the [Datadog Permission API][2]. Find below a list of available permissions.
+
+## General Permissions
+
+General permissions provide the base level of access for your role. [Advanced Permissions](#advanced-permissions) are explicitly defined permissions that augment the base permissions.
+
+| Permission Name | Description                                                                                                                                                                                                                                                                                           |
+| --------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| admin           | This permission gives to the role the ability to view and edit everything in your Datadog organization that does not have an explicitly defined permission. This includes billing and usage, user, key, and organization management. This permission is inclusive of all Standard Access permissions. |
```

Comment:
Maybe it's important to call out the fact that the `admin` permission is the one that determines whether someone can modify roles and permissions, or use the roles and permissions apis?

It's probably important for people to know that if they are going to manage RBAC, they will need this permission. Otherwise they may even risk "locking themselves out" of their RBAC management.